tests/test_match.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import unicode_literals
+
+import unittest
+import sys
+
+import diff_match_patch
+
+if sys.version_info[0] == 3:
+    diff = diff_match_patch.diff
+    diff_bytes = diff_match_patch.diff_bytes
+    match_main = diff_match_patch.match_main
+    match_main_bytes = diff_match_patch.match_main_bytes
+else:
+    diff = diff_match_patch.diff_unicode
+    diff_bytes = diff_match_patch.diff_str
+    match_main = diff_match_patch.match_main_unicode
+    match_main_bytes = diff_match_patch.match_main_str
+
+
+class MatchTests(unittest.TestCase):
+    def test_unicode(self):
```

Comment:
@freakboy3742 It would be great to throw a unicode string containing non-ASCII characters in these assertions.